crates/bin/node/src/initialization/http.rs

```diff
@@ -0,0 +1,70 @@
+#[cfg(feature = "http")]
```

Comment:
put the feature flag around the import of the module in the parent module

```
#[cfg(feature = "http")]
mod http;
```

This way you don't have to repeat it eveywhere haha